Source/AwsCommonRuntimeKit/sdkutils/FileBasedConfiguration.swift

```diff
@@ -0,0 +1,91 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+
+import AwsCSdkUtils
+import struct Foundation.Data
+
+public class FileBasedConfiguration {
+    var rawValue: OpaquePointer
+
+    /// Create a FileBasedConfiguration by merging the configuration from config file and credentials file.
+    /// - Parameters:
+    ///   - configFilePath: (Optional) If a file path is provided use that, otherwise loads config from the default location (~/.aws/config).
+    ///   - credentialsFilePath: (Optional) If a file path is provided use that, otherwise loads config from the default location (~/.aws/credentials)
```

Comment:
If these are nil, won't CRT resolve the path by checking for the corresponding environment variables and then falling back to `~/.aws/...`? Can we update the comments to reflect that

Reply:
Thanks, fixed.